core/src/main/java/org/biojava/bio/symbol/ChunkedSymbolList.java

```diff
@@ -116,4 +121,118 @@ public SymbolList subList(int start, int end) {
       return super.subList(start, end);
     }
   }
+  
+    public void edit(Edit edit) throws IllegalAlphabetException,
+            ChangeVetoException {
+        ChangeSupport cs;
+        ChangeEvent cevt;
+        Symbol[] dest;
+
+        // first make sure that it is in bounds
+        if ((edit.pos + edit.length > length +1 ) || (edit.pos <= 0) || edit.length < 0){
+            throw new IndexOutOfBoundsException();
+        }
+        // make sure that the symbolList is of the correct alphabet
+        if (( edit.replacement.getAlphabet() != getAlphabet()) &&  (edit.replacement != SymbolList.EMPTY_LIST)){
+            throw new IllegalAlphabetException();
+        }
+
+        // give the listeners a change to veto this
+         // create a new change event ->the EDIT is a static final variable of type ChangeType in SymbolList interface
+        cevt = new ChangeEvent(this, SymbolList.EDIT, edit);
+        cs = getChangeSupport(SymbolList.EDIT);
+        synchronized(cs) {
+            // let the listeners know what we want to do
+            cs.firePreChangeEvent(cevt);
+        
+            // now for the edit
+            int posRightFragInSourceArray5 = edit.pos + edit.length - 1;

```

Comment:
Hey look at that, you're right!  Must have been some odd copy and paste error originally.  Feel free to commit a fix in both places and squash on this pull request or I can do it later.